src/main/java/randoop/util/Report.java

```diff
@@ -137,23 +139,24 @@ private void generateCSV(){
             PrintWriter writer = new PrintWriter(new File("objects_report.csv"));
 
             StringBuilder sb = new StringBuilder();
-            sb.append("Classes of objects created");
-            sb.append(',');
-            sb.append("Number of objects created");
-            sb.append(',');
-            sb.append("Number of unique objects manipulated");
-            sb.append('\n');
+            sb.append("Classes of objects created" + ',');
+            sb.append("Number of objects created" + ',');
+            sb.append("Number of unique objects manipulated" + ',');
+            sb.append("Number of objects created (Normalized)" + ',');
+            sb.append("Number of unique objects manipulated (Normalized)" + '\n');
 
             for (Map.Entry<Class<?>, Integer> entry : objectsCreated.entrySet()) {
                 Class<?> objectClass = entry.getKey();
                 Integer value = entry.getValue();
```

Comment:
I'd suggest renaming these variables as their current names do not get us an idea of what they represent.
For example, instead of value, it could be renamed to numberCreatedObjects.
Instead of uniqueValue, it could be renamed to numberUniqueCreatedObjects.

The same applies for the variables normalized_value and normalized_uniqueValue.